coalib/bearlib/languages/definitions/TinyBasic.py

```diff
@@ -0,0 +1,18 @@
+from coalib.bearlib.languages.Language import Language
+
+
+@Language
+class TinyBasic:
+    aliases = 'tb',
+    extensions = '.tb',
+    comment_delimiters = 'REM'
```

Comment:
`comment_delimiters` are a tuple.

Reply:
Can't believe I glanced over that - fixed!